.idea/codeStyleSettings.xml

```diff
@@ -6,6 +6,9 @@
         <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="1000" />
         <option name="JD_ADD_BLANK_AFTER_PARM_COMMENTS" value="true" />
         <option name="JD_ADD_BLANK_AFTER_RETURN" value="true" />
+        <MarkdownNavigatorCodeStyleSettings>
```

Comment:
I think it makes sense to apply the latest code style we have in Spine. So, probably, you'll just need to copy the files from `core-java` repository. And not make any more changes.